### ISSUE TITLE
Add working example of Twitter Skill to docs

### ISSUE THREAD

**Issue by j-greig** [empyrealapp/emp-agents#21]
Would be great if https://emp-agents.empyrealsdk.com/skills/twitter/ had a working example of how to use the skill in the context of an agent - can't find anything for this in the tests folder.

(Also am find the definition of tools vs skills a bit unclear... skill = group of tools?)

**Comment by j-greig** [comment]
PS. Confusion arises because we have :

`docs/docs/en/skills/twitter.md`

and 

`tools/twitter/__init__.py`

**Comment by johnny-emp** [comment]
yes that's right, the docs file is for the documentation (emp-agents.empyrealsdk.com) and the twitter/__init__.py is for the twitter specific tools.  Although after doing some development, it feels much more logical to control the twitter posting explicitly rather than via a tool call.
